Wireframe/index.html

```diff
@@ -8,25 +8,43 @@
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
+      <h1>MODULE 1</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        The purpose of this page is to create a Wireframe template explaining the use of Readme and branching.
       </p>
     </header>
     <main>
       <article>
         <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <h2>Read.Me FIle</h2>
+        <h3>Article summary</summary></h3>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+          A README file is added in a repository to communicate important information about your project
         </p>
-        <a href="">Read more</a>
+        <a href="https://docs.github.com/fr/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes" target="_blank">Read more</a>
+      </article>
+      
+      <article>
+        <img src="placeholder.svg" alt="" />
```

Comment:
I think the idea is to replace any image like 'placeholder'. I can suggest using [unsplash](https://unsplash.com) for free images.